rust-runtime/aws-smithy-types/src/date_time/mod.rs

```diff
@@ -329,6 +330,14 @@ impl Ord for DateTime {
     }
 }
 
+impl Display for DateTime {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let datetime =
+            chrono::NaiveDateTime::from_timestamp_opt(self.seconds, self.subsecond_nanos).unwrap();
+        let formatted = datetime.format("%Y-%m-%dT%H:%M:%S%.fZ");
+        write!(f, "{}", formatted)
```

Comment:
instead of using chrono, we should use the date time support already built into datetime: https://docs.rs/aws-smithy-types/latest/aws_smithy_types/date_time/struct.DateTime.html#method.fmt

make sure to also remove the chrono dependency

Reply:
we'll also need at least one test